packages/harpguru-core/src/components/hole-number-row/utils/get-hole-numbers.test.tsx

```diff
@@ -7,9 +7,15 @@ import { getHoleNumbers } from './get-hole-numbers'
 
 jest.mock('reactn')
 const mockUseGlobal = useGlobal as jest.Mock
+// TODO: Should be shared resource
```

Comment:
Already addressed in a code review above, but this comment needs removing and the code modifying here too.